### PR TITLE
#2539 Post fixes in sales feed of collection

### LINKED ISSUE #2539: Post fixes in sales feed of collection
Post fixes in sales feed of collection
- show default BUY events when you open History
- fix CSS to show it in one line, probably clip long identities

image

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/2523#issuecomment-1066839622_

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #2539
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

Screenshot 2022-03-17 at 1 03 11 PM

### CODE REVIEW

File: components/rmrk/Gallery/History.vue [hunk 1]
```diff
@@ -146,7 +146,7 @@ export default class History extends mixins(ChainMixin, KeyboardEventsMixin) {
   @Prop({ type: Boolean, default: false })
   private readonly openOnDefault!: boolean
   private currentPage = 1
-  private event = 'all'
+  private event: any = this.$t('nft.event.BUY')
```
Comment (roiLeo) [inline components/rmrk/Gallery/History.vue:149]:
```suggestion
  private event: string = this.$tc('nft.event.BUY')
```

**Review by roiLeo — COMMENTED** [review]
Code looks good. missing responsive first column.
Screenshot 2022-03-17 at 09-02-34 Kitty Paradise

I'd like to see if it's possible to use a non-breaking space instead of a css solution.

**Comment by vikiival** [comment]
^-^

**Comment by prachi00** [comment]
> Code looks good. missing responsive first column. Screenshot 2022-03-17 at 09-02-34 Kitty Paradise
> 
> I'd like to see if it's possible to use a non-breaking space instead of a css solution.

@roiLeo can you send the collection url where this is happening? For me it is like this
Screenshot 2022-03-17 at 3 16 44 PM

**Comment by roiLeo** [comment]
> @roiLeo can you send the collection url where this is happening? For me it is like this Screenshot 2022-03-17 at 3 16 44 PM

Tested on: `/rmrk/collection/800f8a914281765a7d-KITTY?tab=history`

Have you tried to resize your browser? You can open dev console on right side and you'll be able to resize

**Comment by prachi00** [comment]
@roiLeo have fixed the issue

**Comment by yangwao** [comment]
pay 100 usd 

let's merge it whenever done )

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 131.9 USD/KSM ~ 0.758 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x65d815842f3af5ab3ee51161f485d91673ff7413c92b3b3bc8e62060a8462247](https://kusama.subscan.io/extrinsic/0x65d815842f3af5ab3ee51161f485d91673ff7413c92b3b3bc8e62060a8462247)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by prachi00** [comment]
> > > Well it's working too bad it's not possible to use a `&nbsp;` char
> > 
> > 
> > use this where? Did not get it
> 
> I was thinking to use it (or another utf8 char) directly in translation file but I don't know if it would work out

umm yeah probably not sure how to do that tho

**Pull request merged**